TeachingRecordSystem/src/TeachingRecordSystem.Core/Models/InductionStatus.cs

```diff
@@ -4,19 +4,19 @@ namespace TeachingRecordSystem.Core.Models;
 
 public enum InductionStatus
 {
-    [InductionStatusInfo("none", requiresStartDate: false, requiresCompletedDate: false)]
```

Comment:
These should all be lower case, like in `MandatoryQualificationSpecialism`. `GetTitle()` will get the title-cased version.